sdk/python/packages/flet-core/src/flet_core/alert_dialog.py

```diff
@@ -407,6 +437,7 @@ def semantics_label(self, value: Optional[str]):
         self._set_attr("semanticsLabel", value)
 
     # clip_behavior
+
```

Comment:
May I know why you added this (space before the comment)?
I personally think we shouldn't modify this :)

Reply:
Ok, I see.
Please can you block them why working on this PR (or at minimum, remove the setting concerning this)? Or you can manually check before commiting.

Please can you revert the changes concerning this blank-lines?

Reply:
I'm already fixing it, but idk how block blank-lines in vs code formatter (i use shift-alt-f)